ui/page/claim/internal/claimPageComponent/internal/channelPage/tabs/contentTab/internal/searchResults.jsx

```diff
@@ -1,29 +1,41 @@
 // @flow
 import React from 'react';
 import ClaimList from 'component/claimList';
-import { DEBOUNCE_WAIT_DURATION_MS, SEARCH_PAGE_SIZE } from 'constants/search';
+import { DEBOUNCE_WAIT_DURATION_MS } from 'constants/search';
+import * as CS from 'constants/claim_search';
 import { lighthouse } from 'redux/actions/search';
 
 type Props = {
   searchQuery: string,
   claimId: ?string,
   showMature: ?boolean,
   tileLayout: boolean,
+  orderBy?: ?string,
   onResults?: (results: ?Array<string>) => void,
   doResolveUris: (Array<string>, boolean) => void,
 };
 
 export function SearchResults(props: Props) {
-  const { searchQuery, claimId, showMature, tileLayout, onResults, doResolveUris } = props;
+  const { searchQuery, claimId, showMature, tileLayout, orderBy, onResults, doResolveUris } = props;
 
+  const SEARCH_PAGE_SIZE = 24;
   const [page, setPage] = React.useState(1);
   const [searchResults, setSearchResults] = React.useState(undefined);
-  const [isSearching, setIsSearching] = React.useState(false);
+  const [isSearchingState, setIsSearchingState] = React.useState(false);
+  const isSearching = React.useRef(false);
   const noMoreResults = React.useRef(false);
+  const sortBy =
+    !orderBy || orderBy === CS.ORDER_BY_TRENDING
+      ? ``
+      : orderBy === CS.ORDER_BY_TOP
+      ? `&sort_by=${CS.ORDER_BY_TOP_VALUE[0]}`
+      : `&sort_by=${CS.ORDER_BY_NEW_VALUE[0]}`;
```

Comment:
I think the default (`!order`) should be the same as `New`?

- If I load https://neko.odysee.tv/@samtime:1?view=content and search, the tab is `New` but the search has no `sort_by` param.
- If I change tab to `Top` and search, there is param as expected
- If I change tab back to `New`, there is `sort_by` param, which is inconsistent with the first try.

Or is "no sort_by" equivalent to "sort_by: release_time`?

Reply:
You're right. I got the defaults mixed up because the default search order is trending (actually relevance).